product_docs/docs/net_connector/8.0.2.1/13_using_a_ref_cursor_in_a_net_application.mdx

```diff
@@ -5,7 +5,7 @@ title: "Using a ref cursor in a .NET application"
 
 <div id="using_a_ref_cursor_in_a_net_application" class="registered_link"></div>
 
-A `ref cursor` is a cursor variable that contains a pointer to a query result set. The result set is determined by executing the `OPEN FOR` statement using the cursor variable. A cursor variable isn't tied to a particular query like a static cursor. You can open the same cursor variable a number of times with the `OPEN FOR` statement containing different queries and each time. A new result set is created for that query and made available by way of the cursor variable. You can declare a cursor variable in two ways:
+A `ref cursor` is a cursor variable that contains a pointer to a query result set. The result set is determined by executing the `OPEN FOR` statement using the cursor variable. A cursor variable isn't tied to a particular query like a static cursor. You can open the same cursor variable a number of times with the `OPEN FOR` statement containing different queries each time. A new result set is created for that query and made available by way of the cursor variable. You can declare a cursor variable in two ways:
```

Comment:
Please check this edit, as I wasn't sure.

Reply:
@moazzumali, can you please confirm?

Reply:
@nidhibhammar Yes. correct.